src/erlang_ls_server.erl

```diff
@@ -51,9 +55,9 @@
 %%==============================================================================
 -spec start_link(module()) -> {ok, pid()}.
 start_link(Transport) ->
-  {ok, Server} = gen_server:start_link(?MODULE, Transport, []),
-  {ok, _} = Transport:start_listener(Server),
-  {ok, Server}.
+  {ok, Pid} = gen_server:start_link({local, ?MODULE}, ?MODULE, Transport, []),
```

Comment:
There should only be a single `erlang_ls_server` at a time, so I added a name to it. This will enable to stop having to pass the `Pid` around for notifications, for example.

Reply:
That was something I noticed as well. 👍